app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt

```diff
@@ -328,6 +335,43 @@ fun DepthSurfaceCircle(
     }
 }
 
+@Composable
+fun ImageCaptureCircle(
+    modifier: Modifier,
+    color: Color,
+    shadowColor: Color,
+    ){
+    Canvas(modifier = modifier.background(shape = CircleShape, color = color)
```

Comment:
This can be used as an icon, so there is no need for the green background color.